src/constraints/constraint_fix_node_pressure_point.jl

```diff
@@ -0,0 +1,60 @@
+#############################################################################
+# Copyright (C) 2017 - 2018  Spine Project
+#
+# This file is part of SpineOpt.
+#
+# SpineOpt is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# SpineOpt is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#############################################################################
+"""
+    constraint_fix_node_pressure_point(m::Model)
+
+Outer approximation of the non-linear terms.
+#Linear apprioximation around fixed pressure points
+"""
+function add_constraint_fix_node_pressure_point!(m::Model)
+    @fetch node_pressure,connection_flow,binary_gas_connection_flow = m.ext[:variables]
+    t0 = startref(current_window(m))
+    m.ext[:constraints][:fix_node_pressure_point] = Dict(
+    (connection=conn,node1=n_orig,node2=n_dest,stochastic_scenario=s,t=t,i=j) => @constraint(
+        m,
+        (sum(
+        connection_flow[conn,n_orig,d,s,t]
+        for (conn,n_orig,d,s,t) in connection_flow_indices(m;connection=conn,node=n_orig,stochastic_scenario=s,direction=direction(:from_node),t=t_in_t(m;t_long=t))
+        )
+        + sum(
+        connection_flow[conn,n_dest,d,s,t]
+        for (conn,n_dest,d,s,t) in connection_flow_indices(m;connection=conn,node=n_dest,stochastic_scenario=s,direction=direction(:to_node),t=t_in_t(m;t_long=t))
+        ))/2 ##### TO DO from node, to node???? for all segments??????

```

Comment:
I'm afraid we'll forget these TODOs as soon as we merge this. Would you consider creating an issue so we remember?